### DISCUSSION TITLE
p/makerustcfwksaibot/

### DISCUSSION THREAD

**Discussion by giscus** [GenshinMinecraft/GMBlog#19] [Announcements]
# p/makerustcfwksaibot/

大爹 Beta API 不计费

https://blogcdn.blog.highp.ing/p/makerustcfwksaibot/

**Comment by 3377** [comment]
这里使用命令来配置，那github里面的config.yaml文件的配置意义何在呢？
最终是执行./amd64-linux还是Rust-TGAIBOT呢，经过测试弄出来了，然后关闭shell窗口机器人就关了？这真的是一言难尽。
